packages/common/src/components/DeactivatedAccount.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import styled from "styled-components";
+import IconButton from "./IconButton";
+import { IconArrowRight } from "hds-react";
+import { fontBold, fontRegular } from "../common/typography";
+
+const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: var(--container-width-xl);
+  margin: 0 auto;
```

Comment:
This whole thing has to be redone after layouts have been merged.
So much copy pasta `margin`, `padding`, `max-width` code.

But if this is merged first we can fixit then.

Reply:
I tried removing as much of the extraneous css-stuff as possible after rebasing it on the current main, so it should look a bit less crowded in that section. Now all of the remaining rules seem logical/justified, to me at least.